phpcs.xml.dist

```diff
@@ -13,7 +13,8 @@
     <!-- Show progress of the run -->
     <arg value="p"/>
 
-    <rule ref="Doctrine"/>
+    <rule ref="DoctrineCodingStandard"/>
```

Comment:
That is quite the BC break - are we sure we have to/want to rename the standard?

Reply:
I run into issues and chose the second solution mentioned:
https://github.com/doctrine/coding-standard/pull/63#issuecomment-404022002

But I admit, I didn't use much time to test how to make it run with the previous namespace.